libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/alert/RouteAlertType.kt

```diff
@@ -7,5 +7,9 @@ enum class RouteAlertType {
     /**
      * Type of the [TunnelEntranceAlert].
      */
-    TunnelEntrance
+    TunnelEntrance,
+    /**
+     * Type of the [BorderCrossingAlert].
+     */
+    BorderCrossing
```

Comment:
Adding new entries to this `public` `enum` will break SemVer in the future.

Reply:
It's a good point. Maybe need to take an approach out of the mapbox-java playbook and define them as strings with annotations

https://github.com/mapbox/mapbox-java/blob/2733e034f33191a68d273fdd4b6b03fa5186be13/services-directions-models/src/main/java/com/mapbox/api/directions/v5/DirectionsCriteria.java#L304

Reply:
I converted those enums to straight-up `Int` constants. `IntDefs` in Kotlin code are not respected by the Android linter, so there would be no use in introducing them.